LMNT/include/lmnt/arm/platform.h

```diff
@@ -11,9 +11,9 @@ extern "C" {
 // #endif
 
 #if defined(__aarch64__) || defined(_M_ARM64)
-#define LMNT_JIT_TARGET_NATIVE LMNT_JIT_TARGET_ARM64
+// #define LMNT_JIT_TARGET_NATIVE LMNT_JIT_TARGET_ARM64
 #elif defined(__ARM_ARCH_7A__)
-#define LMNT_JIT_TARGET_NATIVE LMNT_JIT_TARGET_ARMV7A
+// #define LMNT_JIT_TARGET_NATIVE LMNT_JIT_TARGET_ARMV7A
```

Comment:
do you want to keep these commented out lines?

Reply:
I think so - it's mostly just to remember what the plan was with those - more specifically I do want to keep the `#if defined(ANNOYING_TO_FIND_LATER)` stuff around it, and there might as well be the `#define`s there to make it clear what it was going to do.

Reply:
okeydokey :D